nixos/maintainers/scripts/ec2/amazon-image.nix

```diff
@@ -13,18 +13,20 @@ in {
   # Amazon recommends setting this to the highest possible value for a good EBS
   # experience, which prior to 4.15 was 255.
   # https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/nvme-ebs-volumes.html#timeout-nvme-ebs-volumes
-  config.boot.kernelParams =
-    let timeout =
-      if versionAtLeast config.boot.kernelPackages.kernel.version "4.15"
-      then "4294967295"
-      else  "255";
-    in [ "nvme_core.io_timeout=${timeout}" ];
+  config.boot.kernelParams = let
+    timeout =
+      if versionAtLeast config.boot.kernelPackages.kernel.version "4.15" then
+        "4294967295"
+      else
+        "255";
+  in [ "nvme_core.io_timeout=${timeout}" ];
 
   options.amazonImage = {
     name = mkOption {
       type = types.str;
       description = "The name of the generated derivation";
-      default = "nixos-amazon-image-${config.system.nixos.label}-${pkgs.stdenv.hostPlatform.system}";
+      default =
+        "nixos-amazon-image-${config.system.nixos.label}-${pkgs.stdenv.hostPlatform.system}";
```

Comment:
Did you maybe use the old nixfmt to format this?

Reply:
Oh could be. Does it look wrong?  I can reformat it using the right formatter

Reply:
Done! Thanks for noticing.